File: frontend/src/types/ward.ts
export interface WardLatest {
  ward: number;
  latitude: number;
  longitude: number;
  window_start: string;
  window_end: string;
  avg_fill_level: number;
}

export interface WardHistory {
  window_end: string;
  avg_fill_level: number;
}

File: backend/api/app/models/ward.py
from pydantic import BaseModel  # type: ignore
from datetime import datetime

class WardLatest(BaseModel):
    ward: int
    latitude: float
    longitude: float
    window_start: datetime
    window_end: datetime
    avg_fill_level: float


class WardHistory(BaseModel):
    window_end: datetime
    avg_fill_level: float
